modular_splurt/code/modules/vending/cola.dm

```diff
@@ -1,6 +1,11 @@
-/obj/machinery/vending/cola/Initialize()
-    var/list/extra_products = list(
-        /obj/item/reagent_containers/glass/beaker/waterbottle/large = 5
-    )
-    LAZYADD(products, extra_products)
-    . = ..()
+/* /obj/machinery/vending/cola/Initialize(mapload)
+	var/list/extra_products = list(
+			/obj/item/reagent_containers/glass/beaker/waterbottle/large = 5,
+			/obj/item/reagent_containers/food/drinks/soda_cans/carbonatedcum = 5,
+			/obj/item/reagent_containers/food/drinks/soda_cans/carbonatedfemcum = 5,
+			/obj/item/reagent_containers/food/drinks/soda_cans/blood = 5,
+			/obj/item/reagent_containers/food/drinks/soda_cans/blooddiscrete = 5
+	)
+	LAZYADD(products, extra_products)
+	. = ..()
+Doesn't work, why bother?
```

Comment:
these edits work, the problem is that this file apparently isn't ticked, so it needs to be ticked into the tgstation.dme through vscode

Reply:
ah. Silly me. I spent hours messing with that. I will cry now.